package.json

```diff
@@ -20,7 +20,7 @@
   "author": "Relaying Services Team",
   "license": "MIT",
   "dependencies": {
-    "@rsksmart/rif-relay-client": "https://github.com/infuy/rif-relay-client#revenue-sharing-POC",
+    "@rsksmart/rif-relay-client": "https://github.com/antomor/rif-relay-client#PP-117/transaction-reverted-error",
```

Comment:
We need to change it after the [rif-relay-client PR](https://github.com/infuy/rif-relay-client/pull/10) is approved. Please don't merge it